### DISCUSSION TITLE
Class Code Idea 2

### DISCUSSION THREAD

**Discussion by mraesigler** [sbordyak/TemporalIsolation#3] [Ideas]
Class: System

- emergence (1xn)
- reproductive viability (1xn) = ED+7
- longevity
- sexratio
- % plant tissue available (1xn)
- plant tissue (1xn)

**Comment by mraesigler** [comment]
WINNER

**Comment by imcnulty837** [comment]
Explanation on classes/objects in coding - https://brilliant.org/wiki/classes-oop/
Wikipedia page for object-oriented programming (it's accurate, I checked) - https://en.wikipedia.org/wiki/Object-oriented_programming
Documentation on defining class in Matlab - https://www.mathworks.com/help/matlab/ref/classdef.html
